packages/perseus-editor/src/widgets/python-program-editor.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-useless-escape */
+/**
+ * This editor is for embedding Khan Academy Python programs.
+ */
+
+import {
+    components,
+    Changeable,
+    EditorJsonify,
+} from "@khanacademy/perseus";
+import PropTypes from "prop-types";
+import * as React from "react";
+import _ from "underscore";
+
+import BlurInput from "../components/blur-input";
+
+const {NumberInput} = components;
+
+const DEFAULT_HEIGHT = 400;
+
+type Props = any;
+
+/**
+ * This is the main editor for this widget, to specify all the options.
+ */
+class PythonProgramEditor extends React.Component<Props> {
+    static propTypes = {
+        ...Changeable.propTypes,
+        programID: PropTypes.number,
+        height: PropTypes.number,
+    };
+
+    static widgetName = "python-program" as const;
+
+    static defaultProps: any = {
+        programID: null,
+        height: DEFAULT_HEIGHT,
+    };
```

Comment:
Our convention in Perseus is to define a `DefaultProps` and then use indexed access types to reference the Prop key. 

Eg. 

```suggestion
type DefaultProps = {
    programID: Props["programID"];
    height: Props["heigh"];
}

...

    static defaultProps: DefaultProps = {
        programID: null,
        height: DEFAULT_HEIGHT,
    };
```